makefile

```diff
@@ -6,13 +6,14 @@ INCLUDECADMIUM=-I lib/cadmium/include
 #CREATE BIN AND BUILD FOLDERS TO SAVE THE COMPILED FILES DURING RUNTIME
 bin_folder := $(shell mkdir -p bin/src bin/test)
 build_folder := $(shell mkdir -p build/src build/test/subnet build/test/sender build/test/receiver)
+modified_test_outputs_folder := $(shell mkdir -p test/data/modified)
 
 #TARGET TO RUN EVERYTHING (BIT SIMULATOR + TESTS TOGETHER)
 all: simulator tests
 
 #TARGET TO RUN ONLY BIT SIMULATOR
 simulator: main.o message.o
-	$(CC) -g -o bin/src/ABP build/src/main.o build/src/message.o
+	$(CC) -g -o bin/src/ABP build/src/main.o build/src/message.o include/output_modification/convert_output.hpp 
```

Comment:
You need to supply new header file to all four components pf the project and not just ABP.exe